### PR TITLE
Pip: Fix a code comment

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

### CODE REVIEW

File: analyzer/src/main/kotlin/managers/Pip.kt [hunk 1]
```diff
@@ -271,7 +271,7 @@ class Pip(
             // What pydep actually returns as "repo_url" is either setup.py's
             // - "url", denoting the "home page for the package", or
             // - "download_url", denoting the "location where the package may be downloaded".
-            // So the best we can do is to map this the project's homepage URL.
+            // So the best we can do is to map the project's homepage URL.
```
Comment (sschuberth) [inline analyzer/src/main/kotlin/managers/Pip.kt:274]: Actually, I believe there was just a "to" missing after "this". Or maybe yet better "map it to".

Comment (fviernau) [reply analyzer/src/main/kotlin/managers/Pip.kt:274]: oh true.

Comment (fviernau) [reply analyzer/src/main/kotlin/managers/Pip.kt]: Done.

**Comment by fviernau** [comment]
Merge regardless of appveyor which we're waiting already for for two days. The change is sufficiently covered without appveyor results.

**Pull request merged**